google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -48,6 +48,8 @@
 
 from pikepdf import Pdf
 
+from google.cloud.documentai_toolbox.wrappers.hocr import _Hocr
```

Comment:
Rename to `_hocr` to signal that it is not part of the public API.  Also import the module instead of the class here.